bundles/org.openhab.binding.livisismarthome/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,358 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="livisismarthome"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Switch Channel -->
+	<channel-type id="SwitchActuator">
+		<item-type>Switch</item-type>
+		<label>Switch</label>
+		<description>Switches the current on/off</description>
+		<category>Switch</category>
+	</channel-type>
+
+	<!-- Dimmer Channel -->
+	<channel-type id="DimmerActuator">
+		<item-type>Dimmer</item-type>
+		<label>Dimmer</label>
+		<description>Dimms the connected light</description>
+		<category>Lightbulb</category>
+	</channel-type>
+
+	<!-- RollerShutter Channel -->
+	<channel-type id="RollerShutterActuator">
+		<item-type>Rollershutter</item-type>
+		<label>Blinds Position</label>
+		<description>Controls the blinds</description>
+		<category>Blinds</category>
+		<config-description>
+			<parameter name="invert" type="boolean">
+				<label>Invert Position</label>
+				<description>When invert is true than 0 on LIVISI is UP and 100 is DOWN</description>
+			</parameter>
+		</config-description>
+	</channel-type>
+
+	<!-- BooleanSwitch Channel -->
+	<channel-type id="BooleanStateActuator">
+		<item-type>Switch</item-type>
+		<label>Switch</label>
+		<description>Switches the state on/off</description>
+		<category>Switch</category>
+	</channel-type>
+
+	<!-- ThermostatActuator -->
+	<channel-type id="ThermostatActuator_PointTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature</label>
+		<description>Thermostat target temperature</description>
+		<category>Temperature</category>
```

Comment:
Here you could add default semantic tags:
```suggestion
		<category>Temperature</category>
		<tags>
			<tag>Setpoint</tag>
			<tag>Temperature</tag>
		</tags>
```

Reply:
solved